src/components/common/MaintenanceBar.vue

```diff
@@ -100,11 +100,11 @@ export default {
     justify-content: space-around;
   }
 
-  .maintenance-bar:nth-child(2) {
+  div > div.maintenance-bar:nth-child(2) {
```

Comment:
hmm. why is this necessary @mariopino ?

the css is `scoped` - seems like we could accomplish this somehow with less specificity

Reply:
In screen sizes with max-width less than 1025px, maintenance-bar messagess are shown in the bottom of the page, but they are not stacked, so only the last message is displayed.

So that css code is needed to stack the messages at the bottom. I'm not happy with the solution thought, sure must be a better way to acomplish that.

In the case we want to keep the current behavior (only the last message is displayed in screens <= 1024px width) we can simply delete those css statements.

Reply:
ok cool. i understand. 

i think you are right in trying to solve this stacking problem. but i think we should find a solution that doesn't involve `div > div`. perhaps class names for each banner? or some kind of architectural change might be in order.

Reply:
@jbibla  css was updated in the lasts commits so now it's:

```
 .message-2 {
    bottom: 36px;
  }
  .message-3 {
    bottom: 72px;
  }
  .message-4 {
    bottom: 108px;
  }
  .message-5 {
    bottom: 144px;
  }
```

I would like to use scss/vars to increment the bottom offset automatically in each message but I don't know if it is possible.

Reply:
i don't think we have scss at the moment. i think what you're trying to resolve is a symptom of a bigger issue.

i'll take a look now and try a couple things out.

Reply:
I am probably wrong, but can't you put the messages in a container and then just position them relative to each other?

Reply:
no you're correct. mario and i talked about this yesterday. it will require some modifications. 

i opened https://github.com/luniehq/lunie/issues/2967 as a separate issue.